i18n/i18n_test.go

```diff
@@ -127,3 +127,21 @@ func (s *i18nTestSuite) TestInvalidTextDomainDir(c *C) {
 	var Gtest = G
 	c.Assert(Gtest("singular"), Equals, "singular")
 }
+
+func (s *i18nTestSuite) TestLangpackResolver(c *C) {
```

Comment:
Sorry, this was indeed missing (only one of the two cases covered), I fixed this now.